adapters/handlers/rest/middlewares.go

```diff
@@ -59,6 +62,25 @@ func addHandleRoot(next http.Handler) http.Handler {
 	})
 }
 
+// metaSchemaRegexp is used to intercept requests and redirect them to a dedicated http server independent of swagger
+var metaSchemaRegexp = regexp.MustCompile("/raft/*")
```

Comment:
Happy to change this if we feel it is not 100% correct/we will integrate different stuff in there in the future.

Reply:
It might be a good idea to use `/cluster/*` instead of `/raft/*`. It is really about managing a running cluster These are the first basic endpoints, and they will be extended in the future.